packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/codebuild/start-build-batch.ts

```diff
@@ -0,0 +1,126 @@
+import { Construct } from 'constructs';
+import * as codebuild from '../../../aws-codebuild';
+import * as iam from '../../../aws-iam';
+import * as sfn from '../../../aws-stepfunctions';
+import * as cdk from '../../../core';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for CodeBuildStartBuildBatch
+ */
+export interface CodeBuildStartBuildBatchProps extends sfn.TaskStateBaseProps {
+  /**
+   * CodeBuild project to start
+   */
+  readonly project: codebuild.IProject;
+  /**
+   * A set of environment variables to be used for this build only.
+   *
+   * @default - the latest environment variables already defined in the build project.
+   */
+  readonly environmentVariablesOverride?: { [name: string]: codebuild.BuildEnvironmentVariable };
+}
+
+/**
+ * Start a CodeBuild BatchBuild as a task
+ *
+ * @see https://docs.aws.amazon.com/step-functions/latest/dg/connect-codebuild.html
```

Comment:
```suggestion
 * @see https://docs.aws.amazon.com/codebuild/latest/APIReference/API_StartBuildBatch.html
```
Might be better to provide the API-specific link.